autoload/phpactor/quickfix.vim

```diff
@@ -0,0 +1,13 @@
+function! phpactor#quickfix#build(entries) abort
+    let shouldUseFzf = v:true == get(g:, 'phpactorUseFzfForQuickfix', v:true)
+
+    " fzf.vim is required for fzf#vim#with_preview()
+    if shouldUseFzf && exists("*fzf#complete")
+        call phpactor#fzf#quickfix(a:entries)
+    else
+        call setqflist(values(a:entries))
+        cw " Open only if there is recognized errors only
+    endif
+endfunction
```

Comment:
I like that we separated FZF functionality into a separate file, but not sure it makes sense to have this here? would it make sense to put this back into the main file?

Reply:
The goal is to have one function to build the quickfix so that we don't have to duplicate the logic to choose between fzf or vim quickfix list.
I think the quickfix is a great tool, fzf can be really nice to filter it and we might want it for some others use cases like `find implementation` and all.
I don't know if I make a lot of sense, if I'm not clear enough let me know and I will try again :)

Reply:
So we have a single point to select the "file references" strategy, I think that's good :) Can it be deferentiated in its role from other functions in `phpactor.vim` ?

Reply:
Yes I would love to see the `plugin/phpactor.vim` getting smaller and smaller!
It makes it easier to find things and this file is load when vim is opened.
So the less thing there is in it the less we slow vim when starting even if it's just a bit.
Plus if we want to add a strategy and possibly later another strategy I really think it's better to centralize all the logic in one specific place.